docs/create-packages/Select-assemblies-referenced-by-projects.md

```diff
@@ -9,22 +9,57 @@ ms.topic: conceptual
+
+Finally, some package authors have been confused in the past about NuGet's assembly selection for packages supporting more than one target framework when their package also contains multiple assemblies. If your main assembly supports different target frameworks to your utility assembly, it may not be obvious which `lib/` directories to put all of the assemblies into. By separating each package by assembly name, it's more intuitive which `lib/` folders each assembly should go into. Note, this does not mean having `Package1.net48` and `Package1.net6.0` packages. It means having `lib/net48/Package1.dll` and `lib/net6.0/Package6.0` in `Package1`, and `lib/netstandard2.0/Package2.dll` and `lib/net5.0/Package2.dll` in `Package2`. When Nuget restores a project, Nuget will independently do asset selection for the two packages.
+
+Also note that dependency include/exclude assets is only used by projects using PackageReference. Any project installing your package using `packages.config` will install your dependencies and have its APIs available as well. `packages.config` is only supported by Visual Studio's older .NET Framework project templates. SDK style projects, even those targeting .NET Framework, do not support `packages.config`, and therefore do support dependency include/exclude assets.
 
-## `packages.config` support
+## Not recommended: Multiple assemblies in one package
+
+`PackageReference` and `packages.config` have different features available. Whether you want to support your package consumers who use `PackageReference`, `packages.config`, or both, changes how you must author your package.
+
+NuGet's MSBuild Pack target does not support automatically including project references in the package. There is [an issue on GitHub](https://github.com/NuGet/Home/issues/3891), where community members shared ways they achieved this outcome, which usually involves using `PackagePath` MSBuild item metadata to place files anywhere in the package, as described in [the docs on including content in a package](../reference/msbuild-targets.md#including-content-in-a-package), and using [`SuppressDependenciesWhenPacking` to avoid the project references becoming package dependencies](../reference/msbuild-targets.md#pack-target-inputs). Also consider community developed tools such as [NuGetizer](https://github.com/devlooped/nugetizer).
```

Comment:
Should we really be calling out NuGetizer? 
What warrants NuGetizer's inclusion compared to other alternatives (theoretically) ?

Reply:
I approved, but please make this is addressed before merging.

Reply:
I'm not aware of any other alternatives. I'll change the docs so it says "there exists community created tooling that can do this", but I'm not sure how customers reading these docs are then supposed to discover nugetizer.

Reply:
Also, there's kind of a precedent in the hosting package page of listing a bunch of 3rd party products, some commercial: https://docs.microsoft.com/en-us/nuget/hosting-packages/overview